Explorer/Assets/DCL/PluginSystem/Global/BackpackBusPlugin.cs

```diff
@@ -0,0 +1,24 @@
+using Arch.SystemGroups;
+using DCL.AvatarRendering.Wearables.Helpers;
+using DCL.Backpack;
+using DCL.Backpack.BackpackBus;
+using System;
+
+namespace DCL.PluginSystem.Global
+{
+    public class BackpackBusPlugin : IDCLGlobalPluginWithoutSettings
```

Comment:
What is this plugin for? Are you planning to add anything else? Please keep in mind that plugins are designed for injecting systems, if you need a dependency just create a small container